### PR TITLE
Minor modification in Kubectl cheatsheet.md

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #36640

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -389,7 +389,7 @@ kubectl cluster-info                                                  # Display
 kubectl cluster-info dump                                             # Dump current cluster state to stdout
 kubectl cluster-info dump --output-directory=/path/to/cluster-state   # Dump current cluster state to /path/to/cluster-state
 
-# View existing taints on which exist on current nodes.
+# "View existing taints on which exist on current nodes."
 kubectl get nodes -o=custom-columns=NodeName:.metadata.name,TaintKey:.spec.taints[*].key,TaintValue:.spec.taints[*].value,TaintEffect:.spec.taints[*].effect
```
Comment (mtardy) [inline content/en/docs/reference/kubectl/cheatsheet.md:393]:
```suggestion
# View existing taints on which exist on current nodes.
kubectl get nodes -o='custom-columns=NodeName:.metadata.name,TaintKey:.spec.taints[*].key,TaintValue:.spec.taints[*].value,TaintEffect:.spec.taints[*].effect'
```
mmh I guess what you wanted to do to fix https://github.com/kubernetes/website/issues/36640 was quoting the actual command flag parameters.

Comment (kbhawkey) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @NitishKumar06 , does the suggested change make sense?

Comment (NitishKumar06) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @mtardy Thank you for mentioning this!

Comment (mtardy) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: will you apply this change @NitishKumar06 or should I create a separate issue? :)

Comment (NitishKumar06) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @mtardy Sorry for being late! The changes have been applied now:)

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -389,7 +389,7 @@ kubectl cluster-info                                                  # Display
 kubectl cluster-info dump                                             # Dump current cluster state to stdout
 kubectl cluster-info dump --output-directory=/path/to/cluster-state   # Dump current cluster state to /path/to/cluster-state
 
-# "View existing taints on which exist on current nodes."
+# View existing taints on which exist on current nodes."
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/cheatsheet.md:392]: remove the tailing quote mark?

**Comment by sftim** [comment]
@NitishKumar please edit the title and description for this PR to summarize the change you are making.

We need that detail so we can check that the change you're making matches up with that summary.

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (mtardy) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Hey, no problem but it doesn't seem that you applied the change, it's still missing the singlequotes, and as tengqm stated, you left a double quote at the end!

**Comment by mtardy** [comment]
Your PR contains no changes!

**Comment by NitishKumar06** [comment]
@mtardy The suggested changes were to remove the " at the end of the sentence.  It's applied now!

**Comment by mtardy** [comment]
> @mtardy The suggested changes were to remove the " at the end of the sentence.  It's applied now - [here](https://github.com/kubernetes/website/pull/36645/commits/0388ca5160c2e5ecac1de929331e329846c25542)
> 
> 

Mmmh the " at the end of the sentence was introduced by you in a previous commit...

The actual change asked for the associated issue is suggested here https://github.com/kubernetes/website/pull/36645#discussion_r974516508.

At the moment, your PR has no changes in comparison to upstream.

**Review by mtardy — COMMENTED** [review]
It fixes the issue.
Thanks Nitish :)!

/lgtm

**Comment by reylejano** [comment]
`kubectl get nodes -o='custom-columns=NodeName:.metadata.name,TaintKey:.spec.taints[*].key,TaintValue:.spec.taints[*].value,TaintEffect:.spec.taints[*].effect'` works

/approve

**Pull request merged**